src/sentry/search/snuba/backend.py

```diff
@@ -696,3 +577,151 @@ def snuba_search(
         metrics.timing("snuba.search.num_result_groups", len(rows))
 
     return [(row["issue"], row[sort_field]) for row in rows], total
+
+
+class SnubaSearchBackend(SearchBackend):
+    query_executor = PostgresSnubaQueryExecutor()
```

Comment:
Please don't make this a class attribute. Just instantiate it as a local variable where you use it, this is because the instantiation of this object is very cheap and, by keeping as a class attribute, whoever makes changes to PostgresSnubaQueryExecutor will have to remember the instance can be reused.